airflow/www/app.py

```diff
@@ -80,8 +80,7 @@ def create_app(config=None, session=None, testing=False, app_name="Airflow"):
     api.load_auth()
     api.API_AUTH.api_auth.init_app(app)
 
-    # flake8: noqa: F841
-    cache = Cache(app=app, config={'CACHE_TYPE': 'filesystem', 'CACHE_DIR': '/tmp'})
+    Cache(app=app, config={'CACHE_TYPE': 'filesystem', 'CACHE_DIR': '/tmp'})
```

Comment:
Does the cache still work here - I guess it's referred to somerwhere in the Flask/WSGI app so is protected from GC?

Reply:
Yes. There are still references to this object. GC will not delete this object.